### PR TITLE
Add more region tags 

### PR DESCRIPTION
**Pull request by Carlgo11**

### CODE REVIEW

**Comment by ApCoder123** [comment]
Some of the `investing` entries are duplicated from #6213

File: entries/a/alaskaair.com.json [hunk 1]
```diff
@@ -8,6 +8,14 @@
     },
     "keywords": [
       "transport"
+    ],
+    "regions": [
+      "ca",
+      "cu",
```
Comment (ApCoder123) [inline entries/a/alaskaair.com.json]:
```suggestion
```
Alaska Airlines no longer fly to Cuba - see https://newsroom.alaskaair.com/2017-11-14-Alaska-Airlines-will-discontinue-flying-to-Havana-Cuba

File: entries/t/thalia.de.json [hunk 2]
```diff
@@ -12,31 +11,10 @@
...
-      "hr",
-      "cz",
-      "lt",
-      "ro",
-      "lv",
-      "mt",
-      "si",
-      "is"
+      "li",
+      "lu"
```
Comment (ApCoder123) [inline entries/t/thalia.de.json:17]: Whats the reasoning behind listing only these 4 countries? https://www.thalia.de/shop/hilfe-versand/show/#D2

File: entries/u/ukpostbox.com.json [hunk 3]
```diff
@@ -7,6 +7,9 @@
     ],
     "keywords": [
       "postal"
+    ],
+    "regions": [
+      "gb"
```
Comment (ApCoder123) [inline entries/u/ukpostbox.com.json]: I don't think this entry should have a `gb` region tag based on the website's description:

> Get a personal or business address in the UK regardless of where you are in the world. Read your post online or forward it internationally - a smarter post office.

File: entries/u/uniteddomains.com.json [hunk 4]
```diff
@@ -6,9 +6,12 @@
     "tfa": [
       "totp"
     ],
-    "documentation": "https://help.uniteddomains.com/hc/en-us/articles/115001458813",
+    "documentation": "https://www.uniteddomains.com/twofactorauth/",
     "keywords": [
       "domains"
+    ],
+    "regions": [
+      "us"
```
Comment (ApCoder123) [inline entries/u/uniteddomains.com.json]: This site seems to offer its services internationally

File: entries/t/thalia.de.json (hunk 2 above)
Comment (Carlgo11) [reply entries/t/thalia.de.json:17]: While the site ships to most EU countries, the site is only available in German which would mean that's their primary market. That was my reasoning at least

File: entries/u/ukpostbox.com.json (hunk 3 above)
Comment (Carlgo11) [reply entries/u/ukpostbox.com.json]: Ah my bad :see_no_evil:

**Comment by phallobst** [comment]
It seems to me that the line `uri = URI.parse(uri)` in `validate-urls.rb` causes the checks to fail. The `uri` lambda parameter is not a string, but a URI object and needs no further parsing.

An alternative would be `uri = URI.parse("#{uri}")`, I guess - but I don't know if we have a case that requires this.

**Comment by Carlgo11** [comment]
@phallobst but it's facebook.rb that's failing :confused:

**Comment by phallobst** [comment]
D'oh! I didn't even look and just ran the tests myself, and they failed there...

**Comment by Carlgo11** [comment]
I put a `next if response.header['location'].nil?` in `tests/facebook.rb`. For some reason Facebook isn't returning a Location header for the conradcom page. This seems to only happen on CircleCI and I am thus not able to debug it further.

File: entries/c/copaair.com.json [hunk 5]
```diff
@@ -9,6 +9,15 @@
     },
     "keywords": [
       "transport"
+    ],
+    "regions": [
+      "ar",
+      "br",
+      "ca",
+      "co",
```
Comment (phallobst) [inline entries/c/copaair.com.json:23]:
```suggestion
      "cl",
      "co",
      "cr",
```

File: entries/f/flyfrontier.com.json [hunk 6]
```diff
@@ -8,6 +8,13 @@
     },
     "keywords": [
       "transport"
+    ],
+    "regions": [
+      "do",
+      "sv",
+      "gt",
+      "mx",
+      "us"
```
Comment (phallobst) [inline entries/f/flyfrontier.com.json]: Current list according to [Wikipedia](https://en.wikipedia.org/wiki/List_of_Frontier_Airlines_destinations):
```suggestion
      "ag",
      "bs",
      "bz",
      "cr",
      "cu",
      "do",
      "gt",
      "jm",
      "mx",
      "nl",
      "pr",
      "sv",
      "us"
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Review by phallobst — APPROVED** [review]

File: entries/c/copaair.com.json [hunk 7]
```diff
@@ -9,6 +9,17 @@
...
+    "regions": [
+      "ar",
+      "br",
+      "ca",
+      "cl",
+      "co",
+      "cr",
+      "mx",
+      "pa",
+      "us"
```
Comment (ApCoder123) [inline entries/c/copaair.com.json]:
```suggestion
      "ar",
      "aw",
      "br",
      "bs",
      "bb",
      "bz",
      "bo",
      "ca",
      "cl",
      "co",
      "cu",
      "cr",
      "cw",
      "do",
      "ec",
      "gt",
      "gy",
      "ht",
      "hn",
      "jm",
      "mx",
      "ni",
      "pa",
      "py",
      "pe",
      "pr",
      "sx",
      "sr",
      "sv",
      "tt",
      "us",
      "uy",
      "ve"
```
https://destinationsguide.copaair.com/en/

**Review by ApCoder123 — APPROVED** [review]

**Pull request merged**